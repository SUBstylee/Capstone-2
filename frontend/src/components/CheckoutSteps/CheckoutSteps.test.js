import { render } from '@testing-library/react';
import CheckoutSteps from './CheckoutSteps.jsx'

test("renders CheckoutSteps component without crashing", () => {
    const { getByText } = render(
        <CheckoutSteps />
    );

    expect(getByText("Sign In")).toBeInTheDocument();
    expect(getByText("Shipping")).toBeInTheDocument();
    expect(getByText("Payment")).toBeInTheDocument();
    expect(getByText("Place Order")).toBeInTheDocument();
});

test("matches snapshot", function () {
    const { asFragment } = render(
        <CheckoutSteps />
    );
    expect(asFragment()).toMatchSnapshot();
});