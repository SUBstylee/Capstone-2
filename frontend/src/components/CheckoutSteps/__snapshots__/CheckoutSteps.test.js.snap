// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="CheckoutSteps"
  >
    <div
      class="justify-content-center mb-4 nav"
    >
      <div
        class="nav-item"
      >
        <a
          aria-disabled="true"
          class="nav-link disabled"
          role="button"
        >
          Sign In
        </a>
      </div>
      <div
        class="nav-item"
      >
        <a
          aria-disabled="true"
          class="nav-link disabled"
          role="button"
        >
          Shipping
        </a>
      </div>
      <div
        class="nav-item"
      >
        <a
          aria-disabled="true"
          class="nav-link disabled"
          role="button"
        >
          Payment
        </a>
      </div>
      <div
        class="nav-item"
      >
        <a
          aria-disabled="true"
          class="nav-link disabled"
          role="button"
        >
          Place Order
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
